src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -410,14 +419,19 @@ export default function ConfirmSwapModal({
     return undefined
   }
 
+  const errorType = useMemo(() => {
```

Comment:
what's the goal of memo here?

Reply:
actually i don't need to memoize here, i just wanted the logic to be in a function for readability. i'll update this